components/Chips/src/MDCChipView.h

```diff
@@ -0,0 +1,169 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@interface MDCChipView : UIControl
+
+/*
+ A UIImageView that leads the title label.
+
+ Only visible when the image is non-nil.
+ */
+@property(nonatomic, readonly, nonnull) UIImageView *imageView;
+
+/*
+ A UIImageView that leads the title label. Appears above the imageView.
+
+ Only visible when the chip is selected and the image is non-nil.
+ */
+@property(nonatomic, readonly, nonnull) UIImageView *selectedImageView;
```

Comment:
Annoying question: no `imageView:forState:`? I'm assuming not, but you should probably document. :)

Reply:
+1 To the documentation request. Is the intention to limit the use of focused (tvOS)/highlighted(iOS) state changes to the images?

Reply:
This behaves differently to `imageView:forState:` might. This image view is only visible in the selected state and appears above the standard `imageView`. I'm open to suggestions as to how to improve the doc string and/or property name.

Reply:
So would it be possible for a chip to have both imageViews visible and on-screen at the same time? I think I'm having trouble understanding why there is a separate UIImageView rather than just changing the displayed image.

Reply:
Yes the selectedImageView appears on top of the standard imageView.

Reply:
I've expanded the comment to make it more clear.